docs/source/markdown/podman.1.md

```diff
@@ -66,7 +72,9 @@ If `--hooks-dir` is unset for root callers, Podman and libpod will currently def
 
 Podman and libpod currently support an additional `precreate` state which is called before the runtime's `create` operation.  Unlike the other stages, which receive the container state on their standard input, `precreate` hooks receive the proposed runtime configuration on their standard input.  They may alter that configuration as they see fit, and write the altered form to their standard output.
 
-**WARNING**: the `precreate` hook lets you do powerful things, such as adding additional mounts to the runtime configuration.  That power also makes it easy to break things.  Before reporting libpod errors, try running your container with `precreate` hooks disabled to see if the problem is due to one of your hooks.
+**Warning:**
+
+- The `precreate` hook allows additional mounts to the runtime configuration.  This might make it easy to break things.  Before reporting libpod errors, try running your container with `precreate` hooks disabled to see if the problem is due to one of your hooks.
```

Comment:
```suggestion
- The `precreate` hook allows additional mounts to the runtime configuration.  This might make it easy to break things.  Before reporting libpod errors, try running the container with `precreate` hooks disabled to see if the problem is due to one of the hooks.
```